libvast/vast/option_declaration_set.hpp

```diff
@@ -47,15 +47,15 @@ class option_declaration_set {
 
   /// A declaration of a CLI argument option.
   class option_declaration {
-  public:
+public:
```

Comment:
Access modifiers don't *increase* the indentation level, but they still belong to the same indentation level as their class.